### ISSUE TITLE
[Suggestion] System map rotation control invert toggle

### ISSUE THREAD

**Issue by Kaiser-Indrasil** [tsunamayo/Starship-EVO#3478]
Could we possibly have 2 separate checkboxes in the game options menu for each axis, to invert the system map rotation scheme?
I'm used to rotating in 3D space the other way around both in horizontal and vertical planes and I'm sure there are other people like me, whose habits die hard.

**Comment by tsunamayo** [comment]
Hi, where did this habit come from? I guess another game (maybe I just need to switch my standard control? ) cheers

**Comment by ProPeach** [comment]
A checkbox would be preferable, personally I find the current movement most agreeable. It really just depends on what you're used to in terms of navigating a 3D environment, I use a few different 3D modelling programs and they all have different default settings for things like this with a checkbox for personalisation

**Comment by Kaiser-Indrasil** [comment]
I agree, a couple of check boxes for both horizontal and vertical planes would be perfectly enough.
